README.md

```diff
@@ -1,75 +1,328 @@
+
+```
+kubectl port-forward service/argo-server --namespace cass-operator :2746 
+```
+
+This will enable port forwading on your local machine on an available port:
+
+```
+Forwarding from 127.0.0.1:49181 -> 2746
+Forwarding from [::1]:49181 -> 2746
+```
+
+You should now be able to lauch the Argo web application from a browser on your local machine at http://127.0.0.1:49181 to visually inspect and monitor the testing workflow.
+
+#### Ingress
+
+It's also possible to expose the `argo-server` using Kubernetes ingress configuration.  This approach may be necessary if you're running on an existing cluster or a cluster deployed within a cloud provider.  For more information on Kubernetes ingress configuration, check out the official Kubernetes [documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+
+To use ingress to expose an HTTP service, you'll need an [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) deployed within the cluster.  Depending on the environment, you may already have one available.  If not, there are many available as referenced in the Kubernetes [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) documentation.
+
+With an ingress controller in place, obtain the access address of the ingress load balancer and apply that value to an environment variable like:
+
+```
+export ADELPHI_LB_ADDRESS=127.0.0.1
+```
+
+Next, deploy an ingress configuration like below, which will inject the address set in the environment variable:
 
-It consists of a set of Kubernetes templates orchestrated by an Argo workflow, which allows for easily deploying a test
-scenario into a running k8s cluster. The primary goal is comparing a stable C* version (the "source") against a new or unstable
-version (the "target").
+> :warning: It's important to note that the ingress rule is deployed to the same namespace as Adelphi and the `argo-server` service.
 
-The base workflow launches two C* clusters simultaneously (**source** and **target**), then it runs NoSQLBench on both and finally
-compares the stored data with cassandra-diff.
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: adelphi-ingress
+  namespace: cass-operator
+spec:
+  rules:
+  - host: "argo.$(ADELPHI_LB_ADDRESS).nip.io"
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: argo-server
+            port:
+              number: 2746
+```
+
+You should now be able to lauch the Argo web application from a browser at `http://argo.{ADELPHI_LB_ADDRESS}.nip.io/` to visually inspect and monitor the testing workflow.
+
+> This example uses nip.io to provide wildcard DNS support.  To learn more about nip.io, check out their [documentation](https://nip.io/).
```

Comment:
> This approach may be necessary if you're running on an existing cluster or a cluster deployed within a cloud provider

FYI, port forwarding works just as well on GKE too.
It's great you added this additional information on ingress, but for a short-lived deployment such as Adelphi, I think port-forwarding works fine and it's a consistent, low effort, approach across local and cloud envs.
I'm wondering if we could move this part to the end as an Appendix.

Reply:
I'm restructuring things so that in the core of the getting started flow we'll only discuss port-forward, and I've generalized that text some so it's not "local" specific.

I'll keep some of the content about Ingress in a new Appendix section.

So I think that will cover our bases here.  Although we'll need #94 to make ingress to the results-server easier.  But for now I can include an example of creating the service manually before exposing via ingress, that'll work for now.